docs/source/API/core/Macros.rst

```diff
@@ -4,11 +4,78 @@ Macros
 Version Macros
 --------------
 
-+--------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Macro              | Description                                                                                                                                                                  |
-+====================+==============================================================================================================================================================================+
-| ``KOKKOS_VERSION`` | The Kokkos version; ``KOKKOS_VERSION % 100`` is the patch level, ``KOKKOS_VERSION / 100 % 100`` is the minor version, and ``KOKKOS_VERSION / 10000`` is the major version.   |
-+--------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Version integer macros
+     - Example value assuming Kokkos version 4.2.1
+   * - ``KOKKOS_VERSION``
+     - ``4'02'01``
```

Comment:
```suggestion
     - ``40201``
```

Reply:
Why?

Reply:
That's the literal value, isn't it?

Reply:
From https://en.cppreference.com/w/cpp/language/integer_literal

> Optional single quotes (`'`) may be inserted between the digits as a separator; they are ignored when determining the value of the literal.

Reply:
I now that but I think it's more confusing for the average user looking at this page.

Reply:
I dropped the separators